demos/node-classification/README.md

```diff
@@ -1,90 +1,42 @@
+| [Cluster-GCN][cluster-gcn]               |                |          |                             | ✔️             | ✔                             |
+| [RGCN][rgcn]                             | ✔ (edges)️      |          |                             | ✔️             |                               |
+| [GAT][gat]                               |                |          |                             | ✔️             |                               |
+| [SGC][sgc]                               |                |          |                             | ✔️             |                               |
+| [PPNP][ppnp]                             |                |          |                             | ✔️             |                               |
+| [APPNP][ppnp]                            |                |          |                             | ✔️             |                               |
+| [Attri2Vec][attri2vec]                   |                |          |                             | ✔️             |                               |
+| [GraphSAGE][graphsage]                   | see HinSAGE    |          |                             | ✔️             | ✔ [demo][graphsage-inductive] |
+| [Directed GraphSAGE][graphsage-directed] |                | ✔        |                             | ✔️             | ✔                             |
+| [HinSAGE][hinsage]                       | ✔️              |          |                             | ✔️             | ✔                             |
+| [Node2Vec][node2vec]                     |                |          | ✔ [demo][node2vec-weighted] |               |                               |
+
+How to read this table: ✔ represents native support for graphs of that kind for node classification. Example: suppose graph `G` has node features and multiple edge types; the algorithms that can use all this information directly are RGCN and HinSAGE; one can use more algorithms on `G` by ignoring the edge types and use GCN, Cluster-GCN, etc.
+
+[gcn]: gcn/gcn-cora-node-classification-example.ipynb
+[cluster-gcn]: cluster-gcn/cluster-gcn-node-classification.ipynb
+[rgcn]: rgcn/rgcn-aifb-node-classification-example.ipynb
+[gat]: gat/gat-cora-node-classification-example.ipynb
+[sgc]: sgc/sgc-node-classification-example.ipynb
+[ppnp]: ppnp/ppnp-cora-node-classification-example.ipynb
+[attri2vec]: attri2vec/attri2vec-citeseer-node-classification-example.ipynb
+[graphsage]: graphsage/graphsage-cora-node-classification-example.ipynb
+[graphsage-inductive]: graphsage/graphsage-pubmed-inductive-node-classification-example.ipynb
+[graphsage-directed]: graphsage/directed-graphsage-on-cora-example.ipynb
+[hinsage]: hinsage/README.md
+[node2vec]: node2vec/stellargraph-node2vec-node-classification.ipynb
+[node2vec-weighted]: node2vec/stellargraph-node2vec-weighted-random-walks.ipynb
+
+See [the root README](../../README.md) or each algorithm's documentation for the relevant citation(s).
```

Comment:
Referring back to the main README here is good - we only need the collection of citations in one place.